src/pudl/package_data/meta/datapkg/datapackage.json

```diff
@@ -5475,6 +5475,83 @@
                 "path": "https://creativecommons.org/licenses/by/4.0/"
             }],
             "sources": []
-        }
+        },
+        {
+            "profile": "tabular-data-resource",
+            "name": "eia_epacems_crosswalk",
+            "path": "data/plants_eia.csv",  # NOT SURE
+            "title": "eia_epacems_crosswalk",
+            "encoding": "utf-8",  # NOT SURE
+            "mediatype": "text/csv",
+            "format": "csv",
+            "dialect": {  # NOT SURE ABOUT ANY OF THESE
+                "delimiter": ",",
+                "header": true,
+                "quoteChar": "\"",
+                "doubleQuote": true,
+                "lineTerminator": "\r\n",
+                "skipInitialSpace": true,
+                "caseSensitiveHeader": false
+            },
+            "schema": {
+                "fields": [{
+                        "name": "plant_id_eia",
+                        "type": "integer",
+                        "format": "default",  # NOT SURE WHAT DEFAULT MEANS
+                        "description": "The unique six-digit facility identification number, also called an ORISPL, assigned by the Energy Information Administration."
+                    },
+                    {
+                        "name": "plant_id_epa",
+                        "type": "integer",
+                        "format": "default"
+                    },
```

Comment:
In the original file, this is called the `ORIS code` and the `plant_id_eia` is called the `EIA ORIS`. I haven't seen the EPA ORIS code show up elsewhere, but it's possible that it would be useful for connection with other EPA datasets.